meson.build

```diff
@@ -253,6 +253,10 @@ if get_option('plugin_dell')
   endif
 endif
 
+if get_option('plugin_modem_manager')
+  libmm_glib = dependency('mm-glib')
```

Comment:
Does this not need a very specific minimum version of modem manager?  Or is it just a runtime version check?